src/app/services/util/trainrunsection.helper.ts

```diff
@@ -268,6 +281,16 @@ export class TrainrunsectionHelper {
     trainrunSection: TrainrunSection,
     orderedNodes: Node[],
   ): LeftAndRightTimeStructure {
+    if (orderedNodes.length > 0) {
+      return {
+        leftDepartureTime: orderedNodes[0].getDepartureTime(trainrunSection),
+        leftArrivalTime: orderedNodes[0].getArrivalTime(trainrunSection),
+        rightDepartureTime: orderedNodes[1].getDepartureTime(trainrunSection),
+        rightArrivalTime: orderedNodes[1].getArrivalTime(trainrunSection),
+        travelTime: trainrunSection.getTravelTime(),
```

Comment:
`trainrunSection.getTravelTime()` works only in a non-stop case. For multiple non-stop trainrun sections, the cumulative travel time for these sections is considered.

```suggestion
        travelTime: this.trainrunService.getCumulativeTravelTime(trainrunSection),
```

Before suggested change:

https://github.com/user-attachments/assets/ef0f91ec-aa49-4a6f-91c4-23728cfa8e0b

After:

https://github.com/user-attachments/assets/eb2f8e02-a892-460d-806f-7cebfcb39ecc

Reply:
Good catch! This suggested fix didn't end up working because `getTimeStructure().travelTime` must contain the value edited in the text entry, and that needs to be the non-cummulative time for the perlenkette.

Instead I ended up fixing `PerlenketteSectionComponent.getTravelTime()`.